mllib/src/main/scala/org/apache/spark/ml/clustering/KMeans.scala

```diff
@@ -260,7 +269,8 @@ class KMeans @Since("1.5.0") (
     maxIter -> 20,
     initMode -> MLlibKMeans.K_MEANS_PARALLEL,
     initSteps -> 2,
-    tol -> 1e-4)
+    tol -> 1e-4,
+    distanceMeasure -> DistanceSuite.EUCLIDEAN)
```

Comment:
"DistanceSuite" sounds like a test case, which you can't use here, but, looks like it's an object you added in non-test code. That's confusing.

Reply:
do you have any suggestion about which might be an appropriate name? Thanks.